docs/static/drawio/repo-based-standby.xml

```diff
@@ -0,0 +1 @@
zoBbnQIPv/GkB+R2v+q4ZPD30AZgPljAviWub7DLmTb99Nvi9mOYZ1a8415sXnHPiXG9e1ChUyvXm7HugZoT5R4XzVVyLg+vFHDLNLfRVoUR5b59iJ9IsuE7wmaKPkdB3egSWL3DscYywppaPEPCIfhN1A+Yuy5xz4xLnxrEmUMPKS9Sl+DcgbwXxsQfrJxtRLEx2HTWLTWOE8xjmvKOS/XFIYwZQU8Gfc0a5aug3UKPZVLfA74F6RMh2GHNKGj9vQsUKneGhfcQ8a8m5fonmKeblIW9akE6E50W1Rcrl49zWqG7mfNcjPHyfWGcYH2FuR3KvhFOk9ZNTzcWimouLzw3zBDLrkIKRogX7WJ3GA6HfCtaxMH/tYnxOGgfzA/CcAP+sQD3ZRVg8xK2MY6Y5i/haj/3Fn+LW3+NTU5yPqF/AhsIeA5/rsDbV7zGHTnmSn3qLTfU/jKEkgmxC7ZVq/1md6+Y8ikr5XnhWlH/i/JL0nZsGv1Hy5zEvjH5U5nwH730AnwRjT5phjD7doRwNxEaAL0ZwqE/6rCSs5gj2GaBewZpKyddP8VxIGtGjvJ/k/Wiu344R1OI1sPvC72ItHWJRjLHFvY5Nrq2hf7Sp/kr5jKIGpnJcUJLmwji/JA3oljHl9krKRyWoYyBHBQBWXlMF5Yk5/0Q+U63r3qzdVbgOxC1DDIb3jbko3tPh4zpwyd4XVMN1ML/mmfV+CYfqeumBc78IQZUIguqUUVnHJjhHaU7h88N9KWKcxGyhmA61B+aQwK6Br8F8GPhaqOX/ATR7vj8p3wf1IYP3ebike6BOXuaURyPcOeFJGXv9nTe2OBeWUgzCsaNEtSrALzuo/VwEyue8l+Ugrtep13jFeWeMs13wM1T7wrzeHxCta+fYSTe9bgX7oSKlAN90HFVaK+51pbXZOYn42/tTDmABvgj27bhU+8NcSkK5FvSfmC+XSJ/k/AxsRjeLeYLxM+o7BXUi1cUztHFVN6VcOulMfB6Uk9NovaE+1kifCNuIdQHIQ9lcO4Oc9rwDfmqMNRxf/21tfYx4agfnJPlEjIEy0gShxvUpxJhT7Hei/Hf5gyF3g1WK9Un8+5FOcR1qyBL1Ha7zUCMb7GaMt35yvpy1NNd1ZvQMN2CjA9ACOBYF1fYhpk9IgxFy/kl1G9x/QbkbymdR3QdzxWDvDLRjnFN7Yr2eauFsw2yuNbI+Th2K3Y5U56P3GBS3mt/wYHjO0i+QueUvYcCwa6tybZsivjry+6nYFpgdAuUsrFdEK0Li6hBVtmEl406zw0FYSsjAfquUOuMHFXT3grtZJMzooOfBzESBKwWUokU7LchS5XqdycSMbEBZEdp9YNQRMs0KisRYsVPGnasxBJuvMylHyFLALMNMMVWohJWgXQY/ipqyRrQ7hTO/FBmjJwi5wuK8aFah95c4K4e7XPwKVIGt8CyG6pmMcGOs4OYK7bqDa5jyThw/+36lcX+yFcSdOHgPDBt3MaNxYrC5X++sgahZqbOhLilk9mxgVcKMqyywC6Pw6+ot7T6qq1eKS9a2dINOipE87ehBdQuZRt79I7lsCWHl0k4XVBkSR6sSeQiXPDI+k5iiaAs9CT0TUKMmZAwkrqBjhanCqh4oEZMyrB7uRgLr7tAOKK4ke1hJI0WJlUnw3AFGH4pLuxgkVKtUwcEsIShf2j3x/YrKrFa9nLmBnXKQedHwM1ihTEjVEXwdFLHEVWydo6GcM+QFq0aqnuBuONhBAO+D7APsdAClBRWLrfAEaI01qmCEqKD+uPsHK65VZx0Ev8m6LuyfDG6VlPRdXMWzwNrDz+CB5pZYOzpWujDDDzuFXIww650mY4C4QxQVzMV32JA9KanKCREFZ8MwckMr/GQLiZUzvGa8f9iJUWd9bJ2ef8ReJMHdUKCwxvWuL8qocQYPvY1Cu8wMnKfwnDYWXK9LkXiYlJ614uytjxUNiAQIJC5RFRLHEiwlWOGz5cJ1YSYF1gnc/0mn74BnEmIl6fteu7/IYpd/ymL/Vp0dMNLkNYmvdSb9iJETZuloB0tIu7dgLKhSCpGixLvQKHqm6EuizJ5Pma2S5iKoVK/83jEg8RwsKfIKaYdFCkoasfcaRYEuAdmx4uly9oCzgXVUe8x/Mokp2TR8VgHtJhXfq1L0EdcV8goqVmBbcOcoqlUDd5/gmFhYrYMsPa6BfxtF/3tddsF9rBLVRmHlYD5Jo7qcC3ts0IPgiod9ZBBzUq74if4VdS2ORMW50pL0Jep3DfctWLj3oHSvmJuknBrtzaX8YHoAzVBQ7Rr3uNG1fGu+BGKJjGpQvHeIctdyvW8CY5MSY+uS9RP693rPCWhCF3MAYkWmq4piONS+tQ8XXkToQ6gTwR47nOmkBTFuSKEWhqsadBl4R4iHlX971P+oLsX3wdWCv4O7Tmi3DI4AqiWZohQXVR1FTVBJPFEGHyqFgcM76RzcfQHZrzGpfI1sHkbekrirwv3t3WDFx5a5SoM7QdA/wt/AO0RF/D/NJzhPcImxA8b/zYGZRkv6R6P1uyMzqtL8h9b48zHK1j8a6i8Ozij/aP8fHZ3R/mXP5r+sKfP9cbys4RY/TZn/ikb4fweKg/YvmzL/ZbMze66v281ndv6HZucfm91rfzWmQftVl+XPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcN/8zSX9jc+c6j+q1Ndv0TMSVJT2TR/j5j7O0Ie77fsmTx22X/d7rvH+nl7fLh0/6EDjL8/IqbWUNLfTHZZ/Y+eEdM+E/0z0f/3z5Nr/7mJDkj3Gzz179/1Huv7wb1td/CO/wY=</diagram></mxfile>
```

Comment:
newline :confused: :question:

Reply:
Those files are created by draw.io and it looks like that's how they are generated. I did try adding a newline and it doesn't seem to impact the file being used, but I'm not sure it's worth adding the line to an auto-generated file. What do you think about leaving it as is?

Reply:
that sounds good to me :+1: